To: vim_dev@googlegroups.com
Subject: Patch 7.3.114
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.114
Problem:    Potential problem in initialization when giving an error message
	    early.
Solution:   Initialize 'verbosefile' empty. (Ben Schmidt)
Files:	    src/option.h


*** ../vim-7.3.113/src/option.h	2010-12-02 16:01:23.000000000 +0100
--- src/option.h	2011-02-09 15:37:36.000000000 +0100
***************
*** 854,860 ****
--- 854,864 ----
  # define VE_ONEMORE	8
  #endif
  EXTERN long	p_verbose;	/* 'verbose' */
+ #ifdef IN_OPTION_C
+ char_u	*p_vfile = (char_u *)""; /* used before options are initialized */
+ #else
  EXTERN char_u	*p_vfile;	/* 'verbosefile' */
+ #endif
  EXTERN int	p_warn;		/* 'warn' */
  #ifdef FEAT_CMDL_COMPL
  EXTERN char_u	*p_wop;		/* 'wildoptions' */
*** ../vim-7.3.113/src/version.c	2011-02-09 14:46:58.000000000 +0100
--- src/version.c	2011-02-09 15:46:17.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     114,
  /**/

-- 
From the classified section of a city newspaper:
Dog for sale: eats anything and is fond of children.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
